tools/simulate_geometry.py

```
@@ -0,0 +1,144 @@
+                    "t": t % 100,
+                    "k": -0.5 + (0.1 * random.random()), # Hyperbolic baseline
+                    "epsilon": 0.2
+                }
+            }
+            
+            # 2. Transform into Constellation (Tree)
+            points = []
+            
+            # Extract Paths (Endpoints) - LIMIT to 10
+            if "paths" in snapshot:
+                for i, (path, methods) in enumerate(snapshot["paths"].items()):
```

Ruff (0.14.10)

Loop control variable `methods` not used within loop body

Rename unused `methods` to `_methods`

(B007)